sci-astronomy/wcstools/files/Makefile.am

```diff
@@ -14,7 +14,7 @@ bin_PROGRAMS = \
 	keyhead skycoor subpix xy2sky wcshead conpix gettab newfits getfits \
```

Comment:
This changes existing files used by the previous ebuild, no? We don't want to affect the older version.